assets/src/components/site-scan-results/site-scan-sources-list.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import { Loading } from '../loading';
+import { AMPNotice, NOTICE_TYPE_PLAIN, NOTICE_SIZE_SMALL } from '../amp-notice';
+
+/**
+ * Site Scan sources list component.
+ *
+ * @param {Object} props                         Component props.
+ * @param {Array}  props.sources                 Sources data.
+ * @param {string} props.inactiveSourceNotice    Message to show next to an inactive source.
+ * @param {string} props.uninstalledSourceNotice Message to show next to an uninstalled source.
+ */
+export function SiteScanSourcesList( {
+	sources,
+	inactiveSourceNotice,
+	uninstalledSourceNotice,
+} ) {
+	if ( sources.length === 0 ) {
+		return <Loading />;
+	}
+
+	return (
+		<ul className="site-scan-results__sources">
+			{ sources.map( ( { author, name, slug, status, version } ) => (
+				<li
+					key={ slug }
+					className="site-scan-results__source"
+				>
+					{ name && (
+						<span className={ classnames( 'site-scan-results__source-name', {
+							'site-scan-results__source-name--inactive': [ 'inactive', 'uninstalled' ].includes( status ),
+						} ) }>
+							{ name }
+						</span>
+					) }
+					{ ! name && slug && (
+						<code className="site-scan-results__source-slug">
+							{ slug }
+						</code>
+					) }
+					{ status === 'active' ? (
+						<>
+							{ author && (
+								<span className="site-scan-results__source-author">
+									{ sprintf(
+										// translators: %s is an author name.
```

Comment:
Most of the translators comments in the codebase are formatted as:

```suggestion
										/* translators: %s is an author name. */
```